### ISSUE TITLE
BulkFetch returns an empty list

### ISSUE THREAD

**Issue by ProtoJazz** [pixelunion/elixir-shopifyapi#315]
Am I using this wrong? I cant seem to get a bulk fetch to work when I supply a query. 

Without the query its fine, but I don't really want to query allll the customers. 

When I dumped out the graphql request it was generating it seemed like my query was double encoded. if I decoded it once and ran it again in insomnia it worked fine. 

I couldn't really find any docs so I might just be doing it wrong.
```
ShopifyAPI.GraphQL.BulkFetch.process!(token,
    """
    {
       customers(query:\"tag:some tag OR tag:some other tag\") {
         edges {
           node {
             id
             tags

             }
           }
         }
       }
    """)
```

**Comment by ProtoJazz** [comment]
My bad. I was passing the wrong token and running the query against a store that didn't have those tags. Works fine

**Comment by hez** [comment]
@ProtoJazz thank you for checking out the library. Yeah the docs and examples for BulkFetch are a bit lacking, it is a new addition to this library and to Shopify. We are open to contributions and would love to hear how you are using the library.

Have fun!

**Comment by ProtoJazz** [comment]
I had been using a different elixir + shopify package, but it stopped being updated and the pagination changes pretty much stopped it from working. I found this one searching hex and it seems to be get updated frequently. Just being updated this year is doing better than some libraries, elixir packages aren't as popular as node or ruby ones usually

I switched over but since it was an existing project its kind of a frankenstein. uberauth for login and grabbing and storing the token, then I just make the auth token struct to use this package whenever I need it ` %ShopifyAPI.AuthToken{token: shop.shopify_token, shop_name: shop.shopify_domain}`
